.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 sudo: true
-node_js:
-- stable
+node_js: stable
 script:
+- npm run lint
 - npm test
```

Comment:
Nevermind, `npm test` *does* run lint, so we should just remove it from travis (the current setup just makes travis run lint twice in a row)